tools/fix_schedule_task_refs.py

```
@@ -5,8 +5,8 @@
 ----
 ok-script 的 Windows 计划任务创建时把 -t 参数写成 onetime_tasks 的 1-based 索引
 （如 ``main.py -t 15 -e``）。一旦 onetime_tasks 排序变化，旧索引会指向错误任务。
-应用每次启动时（src/patches/schedule_task_index_sync_patch.py）会自动按缓存里的
-任务名校正索引，无需手动运行本脚本。本工具用于：
+框架（ok-script）现在原生在应用启动时自动校正（见 ok/ui/qt/tasks/schedule_index_sync.py），
```

Ruff (0.16.1)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)